File: Week1/Day4/Exercises/Xp/exercise3.py
from exercise2 import Dog
import random


class PetDog(Dog):
    def __init__(self, name, age, weight, trained=False):
        super().__init__(name, age, weight)
        self.trained = trained

    def train(self):
        print(self.bark())
        self.trained = True
    
    def play(self, *args):
        print("all play together")

    def do_a_trick(self):
        tricks = ["does a barrel roll", "stands on his back legs", "shakes your hand", "plays dead"]
        random_num = random.randint(0, 3)
        print(tricks[random_num])

# chien = PetDog("Lkalb", 3, 30, False)
# # chien.train()
# chien.do_a_trick()

my_dog = PetDog("LKALB", 2, 10)
my_dog.train()
my_dog.play("Buddy", "Max")
my_dog.do_a_trick()

File: Week3/Day2/Exercises/XpGold/exercise3.js
let age = [20,5,12,43,98,55];

let sum = 0;

for (let x in age){
    sum += age[x]
}

console.log(sum)

let max = 0

for (let x in age){
    if (max < age[x])
        max = age[x]
}
console.log(max);

